net-dns/maradns/maradns-2.0.16.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1 systemd toolchain-funcs user
+
+DEADWOOD_VER="3.2.12"
+
+DESCRIPTION="A security-aware DNS server"
+HOMEPAGE="http://www.maradns.org/"
+SRC_URI="http://www.maradns.org/download/${PV%.*}/${PV}/${P}.tar.bz2"
+
+# The GPL-2 covers the init script, bug 426018.
+LICENSE="BSD-2 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="authonly examples ipv6"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}
+	dev-lang/perl"
+
+pkg_setup() {
+	ebegin "Creating group and users"
+	enewgroup maradns 99
```

Comment:
Do you really need the group and users this early? FWICS it isn't used in install phase, so you may delay into `pkg_preinst`. This will save binary package hosts (and failed emerges) from creating them.